app/src/main/java/org/simple/clinic/patient/PatientAddress.kt

```diff
@@ -103,3 +104,7 @@ data class PatientAddress(
     abstract fun getColonyOrVillages(): List<String>
   }
 }
+
+@Entity
+@Fts4(contentEntity = PatientAddress::class)
+data class PatientAddressFts(val uuid: UUID, val colonyOrVillage: String)
```

Comment:
`PatientAddress` has `colonyOrVillage` as nullable. If existing records have a null property, the app will crash. Let's make this nullable as well?

Reply:
good catch